### DISCUSSION TITLE
Investment decisions

### DISCUSSION THREAD

**Discussion by sstroemer** [calliope-project/calliope#480] [General]
## Based on constraint/expression usage
Why are the investment decisions occurring in different amounts of constraints? E.g.
```
storage_cap(_region1_1__csp_) => 4372
storage_cap(_region2__battery_) => 4370

energy_cap(_region1__ac_transmission_region2_) => 8741
energy_cap(_region2__battery_) => 8739

storage_cap(_region1_3__csp_) => 4372
resource_cap(_region1_3__csp_) => 4369
```
I assume this is due to the use of constraints instead of expressions (see also #479), not an "issue", but is there some place I can better understand the mathematical structure?

## Differences in identical techs
Where I am lost is why that also appears to happen for "area" variables (these are probably not "investment" decisions per se, but can carry costs as far as I understand)?

```
resource_area(_region1_1__csp_) => 1807
resource_area(_region1_2__csp_) => 1447
resource_area(_region1_3__csp_) => 1612
```

**Comment by brynpickering** [comment]
I don't really understand what I'm looking at here or what the question is that is being asked, sorry!
